src/styles/blocks/card.scss

```
@@ -0,0 +1,94 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 410px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+    &__image-container {
+      margin: 32px 19px 0;
+    }
+
+    &__image {
+      width: 160px;
+      height: 134px;
+      flex-shrink: 0;
+    }
+
+    &__name {
+      margin: 40px 16px 4px;
```

since we reset margin (as you can see at the comment below) plus add padding to our card we're gonna use only margin-top
```suggestion
      margin-top: 40px;
```